scenario/BulkerScenario.ts

```diff
@@ -8,7 +8,8 @@ import { exp } from '../test/helpers';
 scenario(
   'Comet#bulker > (non-WETH base) all non-reward actions in one txn',
   {
-    filter: async (ctx) => await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}, { network: 'linea-goerli' }]),
+    filter: async (ctx) =>
+      await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}, { network: 'linea-goerli' }, {deployment: 'weth-lrt', network: 'mainnet'}]),
```

Comment:
We suppose, that if WETH deployment was skipped before, we should skip it as well.